.github/labeler.yml

```diff
@@ -35,6 +35,10 @@
 "cranelift:wasm":
   - "cranelift/wasm/**"
   - "cranelift/wasmtests/**"
+  
+"cranelift:area:aarch64":
+  - "cranelift/codegen/src/machinst/*"
```

Comment:
Isn't `machinst` ISA independent? I.e., won't this labeling be wrong as soon as work on supporting other ISAs for the new isel framework picks up?

Reply:
Yes, and we can update the labeling later (it's quick enough to do so). I found it useful to mix both in the meanwhile.

Reply:
Ok, that makes sense :)